ui/app/AppLayouts/ActivityCenter/controls/NotificationAvatar.qml

```diff
@@ -0,0 +1,186 @@
+//     density: 1.25
+//     baseAvatarSize: 36         // base size, multiplied by density
+//     baseBadgeSize: 18
+//     badgeOverlapRatio: 0.22
+//     badgeBgColor: Theme.palette.primaryColor3
+// }
+//
+// NOTES
+// -----
+// * The component’s implicit size can either hug only the avatar or include the
+//   badge extents (see: includeBadgeInImplicit).
+// -----------------------------------------------------------------------------
+
+import QtQuick
+import QtQuick.Effects
+
+import StatusQ.Components
+import StatusQ.Core.Theme
+
+Item {
+    id: root
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // API
+    // ──────────────────────────────────────────────────────────────────────────
+
+    // Content image URL
+    property url  avatarSource: ""
+
+    // Optional badge icon name (empty → hidden)
+    property string badgeIconName: ""
+
+    // When true, render avatar as a circle using a mask.
+    // When false, shows the image provided.
+    property bool  circular: true
+
+    // Density multiplier (acts like a scaling factor)
+    property real  density: 1.0
+
+    // Base sizes (will be multiplied by density)
+    property int   baseAvatarSize: 36
+    property int   baseBadgeSize:  18
+
+    /// Badge background color
+    property color badgeBgColor: Theme.palette.primaryColor3
+
+    // How much the badge overlaps the avatar on bottom-right (0..1)
+    property real  badgeOverlapRatio: 0.25
+
+    // If true, root.implicitWidth/Height include the badge extents.
+    // If false, the implicit size hugs only the avatar.
+    property bool  includeBadgeInImplicit: true
+
+    // When true (default), the avatar area reacts visually to mouse hover and click
+    // and exposes a pointing-hand cursor. Disable if you don’t want the
+    // avatar to be an interactive target.
+    property bool isAvatarClickable: true
+
+    // When true (default), the badge overlay reacts visually to mouse hover and click
+    // and exposes a pointing-hand cursor. Disable if you don’t want the
+    // badge to be clickable or highlight on hover.
+    property bool isBadgeClickable: true
+
+    // Interactions
+    signal avatarClicked()
+    signal badgeClicked()
+
+    QtObject {
+        id: d
+
+        // Dynamic size properties
+        readonly property int avatarSize: Math.round(baseAvatarSize * density)
+        readonly property int badgeSize:  Math.round(baseBadgeSize  * density)
+    }
+
+    // ──────────────────────────────────────────────────────────────────────────
+    // Layout sizing
+    // ──────────────────────────────────────────────────────────────────────────
+    implicitWidth:  root.includeBadgeInImplicit
+                    ? Math.max(avatarImg.width,  badge.x + badge.width)
+                    : avatarImg.width
+    implicitHeight: root.includeBadgeInImplicit
+                    ? Math.max(avatarImg.height, badge.y + badge.height)
+                    : avatarImg.height
```

Comment:
... if only to get some sane sizing by default :) Because now, everytime we add some subcomponent here, this will have to be rewritten